WalletWasabi.Gui/Tabs/SettingsView.xaml

```diff
@@ -64,22 +64,22 @@
               <StackPanel Orientation="Vertical" Spacing="5">
                 <StackPanel Margin="0 10" Orientation="Horizontal" Spacing="5">
                   <ToggleButton IsChecked="{Binding Autocopy}" Content="{Binding Autocopy, Converter={StaticResource BooleanStringConverter}, ConverterParameter=On:Off}" Margin="0 0 10 0" />
-                  <TextBlock VerticalAlignment="Center">Autocopy/Autopaste of bitcoin addresses, bitcoin urls, transaction ids, etc...</TextBlock>
+                  <TextBlock VerticalAlignment="Center">Auto Copy/Paste bitcoin addresses, bitcoin urls, transaction ids, etc...</TextBlock>
                 </StackPanel>
                 <StackPanel Margin="0 10" Orientation="Horizontal" Spacing="5">
                   <ToggleButton IsChecked="{Binding CustomFee}" Content="{Binding CustomFee, Converter={StaticResource BooleanStringConverter}, ConverterParameter=On:Off}" Margin="0 0 10 0" />
-                  <TextBlock VerticalAlignment="Center">Enable manual fee entry.</TextBlock>
+                  <TextBlock VerticalAlignment="Center">Manual fee entry.</TextBlock>
                 </StackPanel>
                 <StackPanel Margin="0 10" Orientation="Horizontal" Spacing="5">
                   <ToggleButton IsChecked="{Binding LurkingWifeMode}" Content="{Binding LurkingWifeMode, Converter={StaticResource BooleanStringConverter}, ConverterParameter=On:Off}" Margin="0 0 10 0" Command="{Binding LurkingWifeModeCommand}" />
                   <TextBlock VerticalAlignment="Center">Lurking Wife Mode, hides sensitive content.</TextBlock>
                 </StackPanel>
               </StackPanel>
             </controls:GroupBox>
-            <controls:GroupBox Title="Screen Lock" TextBlock.FontSize="16" Padding="10" Margin="0 5 10 5">
+            <controls:GroupBox Title="Lock Screen" TextBlock.FontSize="16" Padding="10" Margin="0 5 10 5">
```

Comment:
I don't think it's justified, but I'm fine with either, so ACK (and also ACK for not changing.)